templates/etc/icinga2/conf.d/notifications.conf.j2

```diff
@@ -0,0 +1,202 @@
+/**
+ *  {{ ansible_managed }}
+ *  Configuration from Icinga2 version r2.10.4
+ *
+ *  Icinga2 notifications
+ */
+
+apply Notification "mail-icingaadmin" to Service {
```

Comment:
this whole file is heavily customized to our own needs. any ideas on how we can keep this more generic and globally usable?